base/docs/helpdb/Base.jl

```diff
@@ -4049,13 +4049,6 @@ itself). For matrices, returns an identity matrix of the appropriate size and ty
 one
 
 """
-    parseip(addr)
-
-Parse a string specifying an IPv4 or IPv6 ip address.
```

Comment:
worth tweaking this to be a method-specific docstring for parse?

also needs to be removed from the rst